### PR TITLE
feat(indexer): Hook up api to cli app

### PR DESCRIPTION
**Pull request by roninjin10**
- connect api to cli app so it starts with `indexer-refresh api --config path/to/indexer.toml`

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5942** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5942" target="_blank">Graphite</a> 
    * **PR #5943** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5943" target="_blank">Graphite</a> 
      * **PR #5944** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5944" target="_blank">Graphite</a> 
        * **PR #5945** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5945" target="_blank">Graphite</a> 
          * **PR #5946** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5946" target="_blank">Graphite</a> 
            * **PR #5947** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5947" target="_blank">Graphite</a> 
              * **PR #5948** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948?utm_source=stack-comment).

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5942** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5942" target="_blank">Graphite</a> 
    * **PR #5943** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5943" target="_blank">Graphite</a> 
      * **PR #5944** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5944" target="_blank">Graphite</a> 
        * **PR #5945** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5945" target="_blank">Graphite</a> 
          * **PR #5946** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5946" target="_blank">Graphite</a> 
            * **PR #5947** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5947" target="_blank">Graphite</a> 
              * **PR #5948** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948?utm_source=stack-comment).

File: indexer/cli/cli.go [hunk 1]
```diff
@@ -41,8 +44,16 @@ func runApi(ctx *cli.Context) error {
 	if err != nil {
 		log.Crit("Failed to load config", "message", err)
 	}
-	// finish me
-	return nil
+
+	db, err := database.NewDB(conf.DB.Dsn)
+
```
Comment (hamdiallam) [inline indexer/cli/cli.go:49]: rm newline

File: indexer/cli/cli.go [hunk 2]
```diff
@@ -41,8 +44,16 @@ func runApi(ctx *cli.Context) error {
...
+	if err != nil {
+		log.Crit("Failed to connect to database", "message", err)
+	}
+
+	server := api.NewApi(db.Bridge)
+
```
Comment (hamdiallam) [inline indexer/cli/cli.go:55]: rm newline

**Review by hamdiallam — APPROVED** [review]

File: indexer/cli/cli.go (hunk 1 above)
Comment (roninjin10) [reply indexer/cli/cli.go:49]:
```suggestion
```

File: indexer/cli/cli.go (hunk 2 above)
Comment (roninjin10) [reply indexer/cli/cli.go:55]:
```suggestion
```

**Comment by roninjin10** [comment]
graphite messed up this specific commit for some reason

**Comment by roninjin10** [comment]
messed up graphite here some other prs are going to have to land first before this one actually builds

**Pull request closed without merging**